src/main/resources/magic.xml

```diff
@@ -1046,6 +1046,12 @@
 		</match>
 	</match-list>
 </match>
+<match>
+	<mimetype>application/x-xz</mimetype>
+	<extension>xz</extension>
+	<description>xz compressed data</description>
+	<test offset="1" type="string" comparator="=">7zXZ</test>
```

Comment:
Looks like technically the test string is (https://filext.com/file-extension/XZ):

Identifier:	Hex: FD 37 7A 58 5A 00, ASCII: ý7zXZ 

Mind updating the pull request before I merge?